app/code/Magento/Bundle/Model/Selection.php

```diff
@@ -20,6 +20,8 @@
  * @method \Magento\Bundle\Model\Selection setPosition(int $value)
  * @method int getIsDefault()
  * @method \Magento\Bundle\Model\Selection setIsDefault(int $value)
+ * @method int getWebsiteId()
+ * @method \Magento\Bundle\Model\Selection setWebsiteId(int $value)
```

Comment:
Since we aren't using this approach anymore, this can probably be removed.

Reply:
These methods are still used in another classes.